src/modules/trade/pure/ConfirmDetailsItem/index.tsx

```diff
@@ -30,11 +29,7 @@ export function ConfirmDetailsItem(props: Props) {
           <Row>
             <RowFixed>
               {label && <TextWrapper>{label}</TextWrapper>}
-              {tooltip && (
-                <MouseoverTooltipContent content={tooltip} wrap>
-                  <StyledInfoIcon size={16} />
-                </MouseoverTooltipContent>
-              )}
+              {tooltip && <InfoIcon content={tooltip} />}
```

Comment:
Awesome, I didn't remember this one.

Reply:
Me neither, it was @shoom3301 's suggestion =P